pkg/cmd/system/events.go

```diff
@@ -81,7 +98,19 @@ func Events(ctx context.Context, client *containerd.Client, options types.System
 				}
 			}
 			if tmpl != nil {
-				out := EventOut{e.Timestamp, e.Namespace, e.Topic, string(out)}
+				var data map[string]interface{}
+				err := json.Unmarshal(out, &data)
+				if err != nil {
+					log.G(ctx).WithError(err).Warn("cannot marshal Any into JSON")
+				}
+
+				id := ""
+				_, ok := data["container_id"]
```

Comment:
This should be skipped when `json.Unmarshal` returned an error?

Reply:
Yes, it should. I added an else statement, so now this will only run if `json.Unmarshal` doesn't return an error.